hazelcast-it/distribution-it/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.hazelcast</groupId>
+        <artifactId>hazelcast-it</artifactId>
+        <version>5.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>distribution-it</artifactId>
+    <version>5.1-SNAPSHOT</version>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast</artifactId>
+            <version>5.1-SNAPSHOT</version>
```

Comment:
we should not hard-coded version, please use `${project.version}`. The same for some other dependencies in this pom.xml and in another pom.xml in this PR.

Reply:
Fixed.
Only hardcoded versions left are in the parent sections of the poms.